docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -83,6 +88,18 @@ Leaves the contract without owner. It will not be possible to call
 NOTE: Renouncing ownership will leave the contract without an owner,
 thereby removing any functionality that is only available to the owner.
 
+[.contract-item]
+[[Ownable-transferOwnership]]
+==== `[.contract-item-name]#++transferOwnership++#++(ref self: ContractState, newOwner: ContractAddress)++` [.item-kind]#external#
```

Comment:
```suggestion
==== `[.contract-item-name]#++transferOwnership++#++(ref self: ComponentState<TContractState>, newOwner: ContractAddress)++` [.item-kind]#external#
```
Same question https://github.com/OpenZeppelin/cairo-contracts/pull/767#discussion_r1349038648

Reply:
Let's follow up this in the other comment to keep it centralized.